fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -32,8 +33,42 @@ export default Ember.Route.extend({
     controller.set('rhevTask', model.rhevTask);
     controller.set('openstackTask', model.openstackTask);
     controller.set('cfmeTask', model.cfmeTask);
+    controller.set('deployment', model.deployment);
     controller.stopPolling();
-    controller.startPolling();
+
+    ////////////////////////////////////////////////////////////
+    // NOTE: If an error during a pulp sync occurs, the Katelo::Sync
+    // task scheduled in the Fusor Deploy task tree will throw itself
+    // into a skipped/warning state. This ultimately bubbles, sending
+    // Fusor::Actions::ManageContent into a paused/error state due to
+    // sub-task error.
+    //
+    // We can push the Deployment task into a clean state by triggering
+    // resume on ManageContent. This releases any locks held by the failed
+    // task and allows for redeployment, abondonment, or manual content sync.
+    //
+    // TODO: It's safer for ManageContent to be monitored and resumed
+    // serverside once a given deployment has been initiated.
+    ////////////////////////////////////////////////////////////
+    let contentErrorDiscovered =
+      model.manageContentTask.get('result') === 'error' &&
+      model.manageContentTask.get('state') === 'paused';
+    ////////////////////////////////////////////////////////////
+
+    if(contentErrorDiscovered) {
+      model.deployment.set('has_content_error', true);
+
+      model.deployment.save()
+      .then(() => model.manageContentTask.resume())
+      .then((resumeResult) => this.refresh())
+      .catch((reason) => {
+        console.log(
+          'ERROR: Something broke trying to recover the ManageContentTask');
+      });
+
+    } else if(!model.deployment.get('has_content_error')) {
+      controller.startPolling();
```

Comment:
@isratrade I actually originally had what you're suggesting and stumbled upon that scenario.

```
-> contentErrorDiscovered == true, has_content_error == false (Locked error state)
This is the first state entered when something goes wrong with ManageContent. The task is
in an error/paused state because a sub task (syncrepo) has failed. If this is happens, we
mark the fusor deployment with a content error and resume the task to release any outstanding
locks, then refresh.

-> contentErrorDiscovered == false, has_content_error == true (Unlocked error state)
Because we resumed ManageContent previously, it should now be in a *success/warning*
state, it's been unlocked because of the above branch, but it still has_content_error. It
makes no sense to poll here either since we remain in a suspended "error" state.

Otherwise, poll.
```